D-lama-service/Data/ProjectEntities/Project.cs

```diff
@@ -11,23 +11,30 @@ public class Project : Entity
     {
         [Required]
         public string Name { get; set; }
+
         [Required]
         public string Description { get; set; }
+
         [Required]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public DateTime CreationDate { get; set; }
+
         [Required]
         public DateTime UpdateDate { get; set; }
+
         [Required]
         public bool IsReady { get; set; }
 
         // Required foreign key property
         public int OwnerId { get; set; }
+
         // Required reference navigation to principal
         [JsonIgnore]
         public User Owner { get; set; } = null!;
+
         // Collections navigation containing dependents
         public ICollection<TextDataPoint> TextDataPoints { get; } = new List<TextDataPoint>();
```

Comment:
As we now only have a table for the entity DataPoint (instead of two separate ones for TextDataPoint and ImageDataPoint), we can change this List to DataPoint.
```suggestion
        public ICollection<DataPoint> DataPoints { get; } = new List<DataPoint>();
```